resources/views/vendor/filament-panels/components/logo.blade.php

```diff
@@ -43,26 +43,38 @@
 @endphp
 
 @if ($themeSettings->is_logo_active && $logo)
-    <img
-        src="{{ $logo->getTemporaryUrl(
+    @php
+        $logoUrl = $logo->getTemporaryUrl(
             expiration: now()->addMinutes(5),
-            conversionName: 'logo-height-250px',
-        ) }}"
+            conversionName: $logo->hasGeneratedConversion('logo-height-250px') ? 'logo-height-250px' : '',
+        );
+        $class = ($logo->mime_type == 'image/svg+xml') ? 'h-full max-h-9 w-full' : 'h-full max-h-9 max-w-full';
+    @endphp
+    <img
+        src="{{ $logoUrl }}"
         alt="{{ config('app.name') }}"
         @class([
-            'h-full max-h-9 max-w-full',
+            $class,
```

Comment:
You can drop the variable and do this

```suggestion
            'h-full max-h-9',
            'w-full' => $logo->mime_type == 'image/svg+xml',
            'max-w-full' => $logo->mime_type != 'image/svg+xml',
```